src/lib/token_list_gen.ts

```diff
@@ -128,9 +132,28 @@ export const generateTokenList = async (
     }).sort((a, b) => (a.symbol < b.symbol ? -1 : 1))
     console.log(`List has ${unbridgedL1Tokens.length} unbridged tokens`);
     
+  }
+
+  if(options && options.includeAllL1Tokens) {
+    l2TokenList = l2TokenList.concat(l1TokenList.tokens.filter(
+        (l1TokenInfo) => l1TokenInfo.chainId !== parseInt(l2Network.chainID)
+      )
+      .map((l1TokenInfo)=>{
+        return {
+          chainId: +l1TokenInfo.chainId,
+          name: l1TokenInfo.name,
+          address: l1TokenInfo.address,
+          symbol: l1TokenInfo.symbol,
+          decimals: l1TokenInfo.decimals,
+          logoURI: l1TokenInfo.logoURI
+        }
+    }).sort((a, b) => (a.symbol < b.symbol ? -1 : 1)))
+    
+  } else if(options && options.includeUnbridgedL1Tokens) {
```

Comment:
is there ever a moment when I want to `includeUnbridgedL1Tokens` and `includeAllL1Tokens`?
I think so, but for now we can prob just throw an error if both are set?

Reply:
actually, I think if we just remove the `else` it might work as intended hah
```suggestion
  }
  if(options && options.includeUnbridgedL1Tokens) {
```

Reply:
No I think dont think there's a case for having both because unbridgedL1Tokens should be a sub set of allL1Tokens.

Reply:
Moved the code around a bit so this is more obvious, and check at the start of the function.